gdx/src/com/badlogic/gdx/maps/tiled/BaseTmxMapLoader.java

```diff
@@ -483,7 +483,18 @@ public void run () {
 						throw new GdxRuntimeException(
 							"Error parsing property [\" + name + \"] of type \"object\" with value: [" + value + "]", exception);
 					}
-				} else {
+				} else if (type != null && type.equals("class")) {
```

Comment:
Small thing but you can simplify to "class".equals(type). No need for a null check.